.github/workflows/test-os.yml

```diff
@@ -85,3 +85,28 @@ jobs:
         with:
           name: test-reports
           path: ./bin/testreports
+
+  test-freebsd-amd64:
+    runs-on: macos-12
```

Comment:
Not an issue: is there a reason why it uses macOS?

Reply:
Because the standard Linux instances on GHA do not support nested virtualization